Documentations/set_manufacturing_data.py

```
@@ -0,0 +1,243 @@
+    buffer[4] = year
+    buffer[5] = month
+    buffer[6] = day
+    
+    # Bluetooth name set flag (will be set to 0xFF by firmware)
+    buffer[7] = 0xFF
+    
+    # Production info (44 bytes, padded with spaces)
+    info_bytes = info.encode('ascii')[:44].ljust(44)
+    buffer[8:52] = info_bytes
+    
+    print(f"Writing production data:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)